src/libraries/System.Runtime.Serialization.Formatters/src/System/Runtime/Serialization/FormatterServices.cs

```diff
@@ -16,8 +16,9 @@ public static class FormatterServices
     {
         private static readonly ConcurrentDictionary<MemberHolder, MemberInfo[]> s_memberInfoTable = new ConcurrentDictionary<MemberHolder, MemberInfo[]>();
 
-        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:UnrecognizedReflectionPattern",
-            Justification = "The Type is annotated with All, which will preserve base type fields.")]
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2065:UnrecognizedReflectionPattern",
```

Comment:
What's the difference between `IL2065` and `IL2075`?

Reading https://github.com/dotnet/linker/blob/main/docs/error-codes.md, they sound exactly the same to me.

Reply:
The difference is the input value. The target is always the `this` of a method call - in this case the `GetFields`.
* `IL2065` means - linker was not able to figure out anything about the input value
* `IL2075` means - linker did figure out where the value is from, but it doesn't have the right annotations.

We stopped producing `IL2075` when linker implemented annotation pass through for `Type.BaseType` (in this case it would propagate the `All` annotation to the return value).
But there was a bug, where the read from the array didn't produce an "unknown" value, instead it was treated as "no value at all", which means nothing to do. Once we fixed that, the read from array produces "unknown", which leads to `IL2065`.